api-gateway/src/main/kotlin/com/saveourtool/save/gateway/security/WebSecurityConfig.kt

```diff
@@ -195,7 +196,7 @@ private fun userStatusBasedAuthorizationDecision(
     backendService: BackendService,
     authentication: Mono<Authentication>,
     authorizationContext: AuthorizationContext,
-) = authentication.flatMap { backendService.findByName(it.name) }
```

Comment:
actually, you don't need to get a request to backend here -- authentication.name for basic auth is the save user name here

Reply:
I do need to make a request in order to get `isActive` flag